presto-router/src/main/java/com/facebook/presto/router/cluster/RemoteState.java

```diff
@@ -74,6 +74,7 @@ public synchronized void asyncRefresh()
         if (sinceUpdate.toMillis() > 1_000 && future.get() == null) {
             Request request = prepareGet()
                     .setUri(remoteUri)
+                    .addHeader("Authorization", "Basic " + System.getenv("ROUTER_USER_CREDENTIALS"))
```

Comment:
Hard coding an env variable makes testing this piece of code very difficult. In general it is a good practice to read it via configuration/secret. 
Also, what happens when security is not set up, will this send empty auth header? Shall we make it conditional, send it only when it is present.